scripts/zones/Windurst_Walls/npcs/Greeter_Moogle.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------
+-- Area: Windurst Walls (239)
+--  NPC: Greeter Moogle
+-- !pos NEED POS
```

Comment:
If these greeter moogles have been capped, you can just grab their pos xyz from the npc sql